### PR TITLE
Describe map creation

### PR DESCRIPTION
**Pull request by JingfeiPeng**
This is to address #689 and #690 

Added description of how to create a new map and edit maps.

Added  explanation how to modify the map to allow users to create their own traffic routes. Since previous documentation exists for description of how `route`, `flow` and `mission` works, I added description on how route relates to the map.

Below shows how the doc would look after this change:

image
image
image

### CODE REVIEW

**Review by sah-huawei — APPROVED** [review]
Looks great! :)

File: docs/sim/scenario_studio.rst [hunk 1]
```diff
@@ -93,3 +93,63 @@ And if you have an additional file you wish to turn into a map you can use the c
 
   python3 -m smarts.sstudio.sumo2mesh dataset_public/2lane_sharp/map.net.xml dataset_public/2lane_sharp/map.glb --format=glb
   python3 -m smarts.sstudio.sumo2mesh dataset_public/2lane_sharp/map.net.xml dataset_public/2lane_sharp/map.egg --format=egg
+
+
+The following describes a few basic operations to edit maps.
```
Comment (Gamenot) [inline docs/sim/scenario_studio.rst:98]: Possible subheading:
```suggestion
Create map
---------------
```

File: docs/sim/scenario_studio.rst [hunk 2]
```diff
@@ -93,3 +93,63 @@ And if you have an additional file you wish to turn into a map you can use the c
...
+
+On the top left bar, "file" -> "new network" to create a new map. 
+
+Use shortcut key "e" to change to edge mode. Click "chain" and "two-way" icons located on the far right of top tool bar, shown in the image below:
+
+.. image:: ../_static/chain_two_way.png
+
+Then click on map to start creating new edges and lanes.
+
+Edit existing maps:
```
Comment (Gamenot) [inline docs/sim/scenario_studio.rst:114]: Possible subheading:
```suggestion
Edit existing map
----------------------
```

**Review by Gamenot — APPROVED** [review]

**Comment by Gamenot** [comment]
Good work.

**Pull request merged**